pkg/koordlet/statesinformer/impl/states_noderesourcetopology.go

```diff
@@ -827,6 +815,29 @@ func (s *nodeTopoInformer) calTopologyZoneList(nodeCPUInfo *metriccache.NodeCPUI
 	return zoneList, nil
 }
 
+func (s *nodeTopoInformer) calKubeletAllocatedCPUs(sharePoolCPUs map[int32]*extension.CPUInfo) ([]extension.PodCPUAlloc, error) {
+	// Users can specify the kubelet RootDirectory on the host in the koordlet DaemonSet,
+	// inside koordlet it is mounted to the path /var/lib/kubelet by default.
+	stateFilePath := kubelet.GetCPUManagerStateFilePath(system.Conf.VarLibKubeletRootDir)
+	data, err := os.ReadFile(stateFilePath)
+	if err != nil && !os.IsNotExist(err) {
+		return nil, fmt.Errorf("failed to read state file, err: %v", err)
+	}
+	if err != nil || len(data) <= 0 { // os.IsNotExist(err)
```

Comment:
when will be not exist error

Reply:
noted below:
```go
// NOTE: This happens when the cpu_manager_state or the kubelet allocated cpus are expected to ignore for
// some special cases (e.g. pods' cpuset.cpus are not managed by the koordlet).
```

Reply:
it seems a hack to set wrong`kubelet root dir`.
we need to define which scenario koordinator supports.